.github/workflows/binary-compatibility-checks.yml

```diff
@@ -42,14 +42,14 @@ jobs:
       - name: Report MiMa Binary Issues
         run: |-
           cp .jvmopts-ci .jvmopts
-          sbt "+~ ${{ matrix.scalaVersion }} mimaReportBinaryIssues"
+          sbt "+~ ${{ matrix.scalaVersion }} Test/compile mimaReportBinaryIssues"
```

Comment:
Can we break out Test/compile into a separate step so it is clear it is not mima failing when the code does not compile?

Reply:
Done.